guides/transports.txt

```diff
@@ -0,0 +1,24 @@
+TOPIC
+    transports
+
+DESCRIPTION
+    Bolt uses transports (also known as protocols) to establish a connection
+    with a target in order to run actions on the target. The default transport is
+    SSH, with available transports docker, local, lxd, pcp, podman, remote,
+    ssh, and winrm. Each transport has it's own set of configuration options and
+    defaults, which you can see in our reference documentation at
+    http://pup.pt/bolt-reference.
+
+    You can specify a transport for a target by prepending '<transport>://' to
+    the target's URI. For example, to connect to a target with hostname
+    'example.com' as user 'Administrator' using the WinRM transport, you would
+    pass the following to the target flag:
+        winrm://Administrator@example.com
+    You can also specify a default transport for all targets by passing the
+    '--transport' flag on *nix systems and the '-Transport' flag in Powershell.
+    Finally, you can set the transport for a target in the inventory. For more
+    information about the Bolt inventory, run 'bolt guide inventory'.
```

Comment:
This information is also covered in https://puppet.com/docs/bolt/latest/running_bolt_commands.html . A link to that page might be better.

Reply:
Hm, I don't see the point of the guides being entirely separate from documentation, but rather providing just enough information to hopefully answer user questions without having to go to the docs and then linking to docs if users need to know more. For example targeting flags are also documented https://puppet.com/docs/bolt/latest/running_bolt_commands.html#specify-targets but we opted to include those flags in the `targets` guide.